core/src/test/scala/unit/kafka/server/KafkaConfigTest.scala

```diff
@@ -262,6 +262,8 @@ class KafkaConfigTest {
     props.put(KafkaConfig.ProcessRolesProp, "controller")
     props.put(KafkaConfig.ListenersProp, "PLAINTEXT://127.0.0.1:9092")
     props.put(KafkaConfig.NodeIdProp, "1")
+    props.setProperty(RaftConfig.QUORUM_VOTERS_CONFIG, s"1@localhost:9092")
```

Comment:
This is regular string. String interpolation `s"..."` is not needed. This comment applies to a few places.